package.json

```diff
@@ -12,7 +12,7 @@
     "asn1.js-rfc5280": "^3.0.0",
     "axios": "^1.5.0",
     "big-integer": "^1.6.43",
-    "bignumber.js": "^2.4.0",
+    "bignumber.js": "^6.0.0",
```

Comment:
Current version of bignumber.js is 9.1.2. Should we set this version in package.json?

Reply:
It doesn't need to be set to the latest version, but could be.  I actually think setting this to ">=" would be better, since it would allow for major versions beyond 6.  I'll make that change.

Reply:
Do we really need to support 4 versions? In my opinion during upgrading more interesting for us is confirm of using version 9. It is better for supporting in future. My proposal is to use current latest version (9.1.2) with possibility tu upgrade patch and minor -> ^9.1.2. WDYT?

Reply:
Fine with me, should make testing and support easier - updated.  Thanks again!

Reply:
Thank you